go-controller/pkg/ovn/egressgw.go

```diff
@@ -181,12 +181,6 @@ func (oc *DefaultNetworkController) addPodExternalGWForNamespace(namespace strin
 	}
 	nsUnlock()
 
-	// Include all the dynamic Hop gateway IPs from the policies that apply to this namespace
```

Comment:
why was this removed?

Reply:
Because there is no need for the annotation logic to expose the pod gw IPs from the dynamic hops of the CRs in the namespace annotation. Those are already captured in each CR when there is an update to the CR itself. 
The annotation will still take them into account when deleting GW routes though to ensure it does not delete them by accident.